src/tests/ftest/util/cont_security_test_base.py

```diff
@@ -0,0 +1,319 @@
+            self.log.info("    Unable to destroy container")
+            return False
+        else:
+            return True
+
+
+    def get_container_acl_list(self, pool_uuid, pool_svc, container_uuid,
+                               verbose=False, outfile=None):
+        """Get daos container acl list by daos container get-acl.
+
+        Args:
+            pool_uuid (str): Pool uuid.
+            pool_svc (str): Pool service replicas.
+            container_uuid (str): Container uuid.
+            verbose (bool, optional): Verbose mode.
+            outfile (str, optional): Write ACL to file
+
+        Return:
+            cont_permission_list: daos container acl list.
+
+        """
+        if not secTestBase.check_uuid_format(pool_uuid):
+            self.fail(
+                "    Invalid Pool UUID '%s' provided.", pool_uuid)
+
+        if not secTestBase.check_uuid_format(container_uuid):
+            self.fail(
+                "    Invalid Container UUID '%s' provided.", container_uuid)
+
+        result = self.daos_tool.container_get_acl(pool_uuid, pool_svc,
+                                    container_uuid, verbose, outfile)
+
+        cont_permission_list = []
+        for line in result.stdout.splitlines():
+            if not line.startswith("A:"):
+                continue
+            elif line.startswith("A::"):
+                found_user = re.search(r"A::(.+)@:(.*)", line)
+                if found_user:
+                    cont_permission_list.append(line)
+            elif line.startswith("A:G:"):
+                found_group = re.search(r"A:G:(.+)@:(.*)", line)
+                if found_group:
+                    cont_permission_list.append(line)
+        return cont_permission_list
+
+
+    def compare_acl_lists(self, get_acl_list, expected_list):
+        """Compares two permission lists
+
+        Args:
+            get_acl_list (str list): list of permissions obtained by get-acl
+            expected_list (str list): list of expected permissions
+
+        Returns:
+            True or False if both permission lists are identical or not
+        """
+        self.log.info("    ===> get-acl ACL:  %s", get_acl_list)
+        self.log.info("    ===> Expected ACL: %s", expected_list)
+
+        if len(get_acl_list) == len(expected_list):
+            for element in get_acl_list:
+                if element in expected_list:
+                    continue
+                else:
+                    return False
+            return True
+        return False
+
+
+    def cleanup(self, types):
+        """Removes all temporal acl files created during the test.
+
+        Args:
+            types (list): types of acl files [valid, invalid]
+
+        """
+        for t in types:
+            get_acl_file = "acl_{}.txt".format(t)
+            file_name = os.path.join(self.tmp, get_acl_file)
+            cmd = "rm -r {}".format(file_name)
+            #TODO: Replace the following line with the commented one
```

Comment:
(lint) Replace the following line with the commented one

Reply:
Forgot to remove this TODO comment. Done